src/braket/pennylane_plugin/braket_device.py

```diff
@@ -162,6 +171,8 @@ def _pl_to_braket_circuit(
         )
         if compute_gradient:
             braket_circuit = self._apply_gradient_result_type(circuit, braket_circuit)
+        elif isinstance(circuit.observables[0], ShadowExpvalMP):
+            return braket_circuit
         else:
```

Comment:
Just a matter of taste, but
```suggestion
        elif not isinstance(circuit.observables[0], ShadowExpvalMP):
```